Next.s 9 | API | graphql - Can't resolve 'fs' in /api/index.ts

# Examples bug report

## Example name

api-routes-graphql

## Describe the bug

When I changed the name of the GraphQL API file from /api/graphql.js to /api/index.js, I noticed that the following error appears when building (after executing next build)

> Failed to compile.
./node_modules/busboy/lib/main.js
Module not found: Can't resolve 'fs' in '/Users/mzygmunt/apps/next.js/examples/api-routes-graphql/node_modules/busboy/lib'
Build error occurred
Error: > Build failed because of webpack errors
    at build (/Users/mzygmunt/apps/next.js/examples/api-routes-graphql/node_modules/next/dist/build/index.js:6:847)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! api-routes-graphql@1.0.0 build: `next build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the api-routes-graphql@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mzygmunt/.npm/_logs/2019-07-27T09_46_44_792Z-debug.log

## To Reproduce

1. Clone example api-routes-graphql
2. Change filename /api/graphql.js to /api/index.js
3. Execure npm run build
4. See error

## Expected behavior

After changing the file name, API should be available at /api.

## Screenshots

![image](https://user-images.githubusercontent.com/13297558/61993005-00516d80-b066-11e9-8237-279dc505c126.png)

## System information

- OS: macOS
- Browser (if applies) chrome
- Version of Next.js: 9.0.2 - 9.0.3-canary.3

--- a/packages/next/build/entries.ts
+++ b/packages/next/build/entries.ts
@@ -68,7 +68,7 @@ export function createEntrypoints(
   Object.keys(pages).forEach(page => {
     const absolutePagePath = pages[page]
     const bundleFile = page === '/' ? '/index.js' : `${page}.js`
-    const isApiRoute = bundleFile.match(API_ROUTE)
+    const isApiRoute = page.match(API_ROUTE)
 
     const bundlePath = join('static', buildId, 'pages', bundleFile)
 

--- a/test/integration/api-support/pages/api/index.js
+++ b/test/integration/api-support/pages/api/index.js
@@ -1,3 +1,6 @@
+/* eslint-disable-next-line */
+import fs from 'fs'
+
 export default (req, res) => {
   res.send('Index should work')
 }

